sdk/eventgrid/azure-eventgrid/swagger/README.PYTHON_T2.md

```diff
@@ -0,0 +1,21 @@
+# Azure EventGrid Client Generation for Python
```

Comment:
I'm paranoid since Mayuri was not pleased when the SB swagger founds its way into Master; @lmazuel Is having this generation readme here kosher to your eyes?

Reply:
agree - this is misleading - changing it